client/src/Service/Audit/AuditEvents.php

```diff
@@ -251,11 +253,35 @@ public function csvUploaded(
         return $event + $this->baseEvent(AuditEvents::EVENT_CSV_UPLOADED);
     }
 
+    /**
+     * @param string       $trigger      , what caused the event
+     * @param Organisation $organisation , the org the user has been removed from
+     * @param User         $currentUser  , return the logged in user
+     *
+     * @return array|string[]
+     *
+     * @throws \Exception
+     */
+    public function orgCreated(string $trigger, User $currentUser, Organisation $organisation): array
+    {
+        $event = [
+            'trigger' => $trigger,
+            'current_user' => $currentUser->getEmail(),
```

Comment:
I think this is more like `created_by` as in either case it will be someone uploading a CSV or clicking a button.